src/tests/stringy.spec.ts

```diff
@@ -53,4 +59,63 @@ describe('Stringy methods', () => {
       expect(result).to.equal(true);
     });
   });
+
+  describe('isSuffix', () => {
```

Comment:
Codacy has a fix for the issue: [Replace `'isSuffix'` with `"isSuffix"`](https://app.codacy.com/gh/VictorMaria/Strings-and-Arrays/pullRequest?prid=9807711)
```suggestion
  describe("isSuffix", () => {
```